exporter/zipkinexporter/zipkin.go

```diff
@@ -86,25 +92,37 @@ func zipkinEndpointFromNode(node *commonpb.Node, serviceName string, endpointTyp
 	//      "port": "9000",
 	//      "serviceName": "backend",
 	// }
-	attributes := node.Attributes
 
 	var ipv4Key, ipv6Key, portKey string
 	if endpointType == isLocalEndpoint {
-		ipv4Key, ipv6Key, portKey = "ipv4", "ipv6", "port"
+		ipv4Key, ipv6Key, portKey = zipkin.LocalEndpointIPv4, zipkin.LocalEndpointIPv6, zipkin.LocalEndpointPort
 	} else {
-		ipv4Key, ipv6Key, portKey = "zipkin.remoteEndpoint.ipv4", "zipkin.remoteEndpoint.ipv6", "zipkin.remoteEndpoint.port"
+		ipv4Key, ipv6Key, portKey = zipkin.RemoteEndpointIPv4, zipkin.RemoteEndpointIPv6, zipkin.RemoteEndpointPort
 	}
 
 	var ip net.IP
 	ipv6Selected := false
-	if ipv4 := attributes[ipv4Key]; ipv4 != "" {
-		ip = net.ParseIP(ipv4)
-	} else if ipv6 := attributes[ipv6Key]; ipv6 != "" {
-		ip = net.ParseIP(ipv6)
-		ipv6Selected = true
+	if ipv4, ok := attributes[ipv4Key]; ok {
```

Comment:
Would this complex if statement be better as a function that could return early?

Reply:
The whole code for the zipkin receiver seems a good candidate for a refactor...